mtcpclient/processing.go

```diff
@@ -0,0 +1,69 @@
+package mtcpclient
+
+import (
+	"fmt"
+	"math"
+	"time"
+
+	"github.com/dachad/tcpgoon/tcpclient"
+)
+
+type groupOfConnections struct {
```

Comment:
you don't need an struct just to wrap an standard type. You can directy "type groupOfConnections []tcpclient.Connection". See http://blog.riff.org/2014_07_19_golang_fun_adding_methods_to_primitive_types

Reply:
thks!